competition/Ball ballToRemove
---
bool success
